int8 AngleV
int8 AngleH
float64 TimeF
---
uint8 Data